### PR TITLE
gluon-config-mode-core: hand over more info to the reboot screen

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+local site = require 'gluon.site_config'
+local util = require 'luci.util'
+local gluon_luci = require 'gluon.luci'
+local sysconfig = require 'gluon.sysconfig'
+local pretty_hostname = require 'pretty_hostname'
 
-return function () luci.template.render_string(i18n.translate('gluon-config-mode:reboot')) end
+local uci = luci.model.uci.cursor()
+
+local pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
```
Comment (NeoRaider) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: You can't do this, gluon-config-mode-core must not expect that fastd is installed on the router. We already have gluon-config-mode:pubkey for dealing with VPN-specific output.

File: package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua [hunk 2]
```diff
@@ -1,3 +1,27 @@
+local cbi = require 'luci.cbi'
```
Comment (NeoRaider) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: cbi is unused, remove this line

File: docs/site-example/i18n/en.po [hunk 3]
```diff
@@ -32,10 +32,10 @@ msgstr ""
 
 msgid "gluon-config-mode:reboot"
 msgstr ""
-"<p>The node is currently rebooting and will try to connect to other "
-"nearby Freifunk nodes after that. "
+"<p>Your node <em><%= escape(hostname) %></em> is currently rebooting and will "
+"try to connect to other nearby Freifunk nodes after that. "
 "For more information on the Freifunk Duckburg community, have a look at "
-"<a href=\"https://entenhausen.freifunk.net/\">our homepage</a>.</p>"
+"<a href=\"https://entenhausen.freifunk.net/#<%= sysconfig.primary_mac %>\">our homepage</a>.</p>"
```
Comment (NeoRaider) [inline docs/site-example/i18n/en.po]: These changes don't make sense. Please just don't change the examples at all, at least not in this PR.

File: package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua (hunk 1 above)
Comment (rubo77) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: Do I have to remove the line `local util = require 'luci.util'` then as well?

**Comment by rubo77** [comment]
>Please just don't change the examples at all, at least not in this PR.
why not? Showing the hostname on the reboot screen would be very useful if you flash more routers in the same browser in more windows. and it does no harm either. And a change in the doc files would not be possible before this PR would be accepted, so I think they belong together.

File: package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua (hunk 1 above)
Comment (NeoRaider) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: Yes, that would make sense.

**Comment by NeoRaider** [comment]
I'm fine with keeping the hostname changes.

File: package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+local uci = luci.model.uci.cursor()
+
+local hostname = pretty_hostname.get(uci)
+local contact = uci:get_first('gluon-node-info', 'owner', 'contact')
+
+local msg = i18n.translate('gluon-config-mode:reboot')
+
+return function ()
+  luci.template.render_string(msg, {
+    pubkey = pubkey,
```
Comment (NeoRaider) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: This line must be removed, too.

Comment (NeoRaider) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: Yes.

Comment (rubo77) [inline package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/reboot/0900-msg-reboot.lua]: done

**Pull request merged**